tasks/build/package_scripts/post_install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+user_check() {
+  getent passwd "$1" > /dev/null 2>&1
+}
+
+user_create() {
+  # Create a system user. A system user is one within the system uid range and
+  # has no expiration
+  useradd -r "$1"
+}
+
+if ! user_check "kibana" ; then
```

Comment:
I see the `"kibana"` username literal in a few places. Would it be possible to define it in one file and source that into all these files? It may not be.